fbw-a32nx/src/systems/instruments/src/EFB/Ground/Pages/Payload/A320_251N/A320Payload.tsx

```diff
@@ -138,6 +144,15 @@ export const A320Payload: React.FC<A320Props> = ({
         COMPLETED: 6,
     };
 
+    const dispatch = useAppDispatch();
+
+    useEffect(() => {
+        if (simbriefDataLoaded === true && simbriefWeightsImport === 'ENABLED' && payloadImported === false) {
+            setSimBriefValues();
+            dispatch(setPayloadImported(true));
+        }
+    }, []);
+
```

Comment:
It is probably possible to have the simbrief components be common between both the A320 and A380, unless there is some blocking thing that I have forgotten about since it has been a while, in which case disregard. 

Is it possible to refactor this to reside in the common Payload.tsx element?

Reply:
Uhm ... I honestly don't know 😅 
I've gone by the Patterns/Stuff seen in the existing Code. And in none of the "Common/Base Page including AC specific Pages" Structures I've seen, an useEffect was used on the Common Page.

But I surely can test it later! ("What can possibly go wrong?" ^^)

Reply:
Looked again at it: doing it on the Common Page has the same Problem which lead me to this Approach in the first Place! The Code/Logic to import SimBrief's Payload and Fuel are handled by Functions in the specific Pages (setSimBriefValues() and handleFuelAutoFill() ) which I need to call. Which call other Functions on that (specific) Page. And I just don't know how to call a Function of Page from somewhere else (and did not see any Pattern in that Direction).